backend.sh

```diff
@@ -29,8 +29,6 @@ install() {
   # Install requirements for Django Server or tox.
   if [ "$cli_flag" = "development" ]; then
     pip install -r ./requirements/local.txt
-  elif [ "$cli_flag" = "test" ]; then
-    pip install -r ./requirements/test.txt
```

Comment:
I'm not sure this would work without the testing dependencies being cached on Travis.

Reply:
How do we check this? The PR passes Travis.

Reply:
I don't think this would be necessary for the backend tests, as [the tox config](https://github.com/cfpb/cfgov-refresh/blob/master/tox.ini#L29) also installs this file. So unless there's something in this file that's needed for frontend, it can probably be removed.

Reply:
Yeah, [Travis doesn't use the `backend.sh` script to install test-running dependencies at all](https://github.com/cfpb/cfgov-refresh/blob/master/travis_install.sh#L13). Travis installs `requirements/travis.txt`, which includes the tools necessary to run the tests (like `tox`). Tox then installs`requirements/test.txt`, which includes the things required by the tests (libraries like `mock` and `model_mommy`).

Reply:
ok, thanks @willbarton and @chosak.